packages/apollo-link-state/src/index.ts

```diff
@@ -19,7 +25,21 @@ export type WriteData = {
 
 export type ApolloCacheClient = ApolloCache<any> & WriteData;
 
-export const withClientState = resolvers => {
+export const withClientState = ({ resolvers, defaults, cache }) => {
+  if (!resolvers) {
+    throw new Error(
+      `Resolvers are required to initialize the state link. Please see [DOCS link] for details`,
```

Comment:
Hmmm, sounds like you don't think we should be able to initialize without resolvers. Also, do we have a test for what gets printed if you do `withClientState()`? I feel like that will result in some jank transpilation error.

Reply:
Let's find out!